src/components/Layout/Header/Header.module.css

```diff
@@ -1,3 +1,152 @@
 .root {
-  border: 1px solid red;
+  /*   border: 1px solid red; */
+  height: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  position: sticky;
+  top: 0;
+  left: 0;
+  background-color: rgba(0, 0, 0, 0.088);
+  font-family: "Avenir Light";
+}
+
+.root > .headerContainer {
+  position: relative;
+  width: 900px;
+}
+
+.logo_container {
+  z-index: 3;
+  margin-right: auto;
+  position: relative;
+}
+
+nav {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  transition: all 0.5s;
```

Comment:
It's better to give a specific animation instead of animating everything, like: 

`transition: position 0.5s;`